roles/grafana_cloud_operator/tasks/delete_grafana_oncall_hub_spoke.yml

```diff
@@ -34,6 +34,10 @@
     kind: ManagedCluster
   register: managed_clusters_raw
 
+- name: Initialize managed_clusters as an empty list
```

Comment:
Add a comments on each code block why you need this , like first initialize list then extract names of managed clusters .. currently code is not very understandable